### ISSUE TITLE
Consider removing undo/redo buttons

### ISSUE THREAD

**Issue by misaugstad** [ProjectSidewalk/SidewalkWebpage#1264]
As @adash12 noted [here](https://github.com/ProjectSidewalk/SidewalkWebpage/issues/772#issuecomment-315770351), 16.9% of users have used the undo button, and only 4.4% have used the redo button. Although this is actually a significant share of users, I still don't think it is worth maintaining this feature for the following reasons:

1. The undo button gives almost no better functionality than just deleting a label. The redo gives a _slight_ advantage over re-placing a label and marking severity, but only 4.4% of users have _ever_ used that (and we don't know how many use it frequently).
2. It is a feature that I expect is easier to break than other features. I get this feeling because it keeps this stack of actions, but we are constantly making changes to how the tool works, and I can see us easily not remembering to add things to this stack, or not appropriately updating things that need to be updated when we take something off this stack.
3. Testing that this feature still works is necessary for most of the changes to the front-end. Any changes to labeling or logging requires testing this feature. And testing this feature works isn't simple. We need to place different labels in different situations and try undo/redo, and then also check in the database that they are marked as deleted or not appropriately.
4. Minor point: There is visual clutter on our very cluttered tool for this not-too-useful feature.

In conclusion, maintaining this feature seems like more overhead than it is worth. And once we manage to break it somehow, I think that fixing it will be way more work than the feature itself is worth. So I advocate just removing the feature going forward :)

However, if others find the undo/redo more useful, please let me know!

@jonfroehlich thoughts?

**Comment by jonfroehlich** [comment]
Agree.

**Comment by misaugstad** [comment]
Cool, @aileenzeng when you get a chance, can you mock up what the interface would look like without the undo/redo?

**Comment by jonfroehlich** [comment]
Thanks Aileen. Can I see with right-aligned buttons as well?

**Comment by aileenzeng** [comment]
Yep! 

| Original Zoom | Small Zoom |
| -------------- | ------------ |
| ![image](https://user-images.githubusercontent.com/25534091/41756696-afcce236-7592-11e8-8ba4-5339cfad0fe9.png) | ![image](https://user-images.githubusercontent.com/25534091/41756697-b2ceb554-7592-11e8-8596-738191c38891.png) |

I felt like the spacing other label/zoom buttons feels a bit wide - it feels better for the original zoom buttons though.

**Comment by jonfroehlich** [comment]
Thanks! You're right that right-alignment doesn't work! :-/

I like this one the best: https://user-images.githubusercontent.com/25534091/41754745-8ea2c86e-7588-11e8-806a-f2694ecdfb34.png

Nice job.

**Comment by aileenzeng** [comment]
Agreed!

**Comment by misaugstad** [comment]
Nice! I liked that one and the left-aligned small zoom, so let's go with the image we all liked!
